docs/docs/theory_2.md

```diff
@@ -44,10 +44,94 @@ $$
 \rho_{ij}(t) = e^{-\frac{i}{\hbar}\left(\lambda_i-\lambda_j\right) t}\rho_{ij}(0)
 $$
 
-This way we can see that the equations are completely decoupled. Coefficients on the diagonal of the density matrix don't change, while off-diagonal coefficients gain a phase factor at a constant rate that is dependent on the differences between the Hamiltonian eigenvalues. This method gives us the exact evolution of the system and perfectly preserves unitarity. The downside of it is that it requires a full diagonalization of the Hamiltonian. However, many spin systems that we are interested in are relatively small, and one single diagonalisation for each of them isn't a big deal. Cheaper, more approximate methods might be implemented in the future, but at the moment, MuSpinSim can easily carry out calculations on systems that don't exceed nine or ten spins on a laptop in a few minutes.
+This way we can see that the equations are completely decoupled. Coefficients on the diagonal of the density matrix don't change, while off-diagonal coefficients gain a phase factor at a constant rate that is dependent on the differences between the Hamiltonian eigenvalues. This method gives us the exact evolution of the system and perfectly preserves unitarity. The downside of it is that it requires a full diagonalization of the Hamiltonian. However, many spin systems that we are interested in are relatively small, and one single diagonalisation for each of them isn't a big deal. For a cheaper, more approximate method see [Celio's Method](#celios-method).
 
 > **For developers:** time evolution of a system is handled by the `.evolve()` method of the `Hamiltonian` class.
 
+#### A faster method
+
+For cases when $\frac{B}{T} \rightarrow 0$, for instance when we are simulating a zero field experiment or one where we take $T\rightarrow = \infty$, we no longer need the density matrix and automatically employ a faster method. To do we can first note that we will be dealing with a spin-polarised muon with various other unpolarised spins. The density matrix at $t=0$ for the muon polarised along ${\hat n}$ can be written as
+
+$$
+ \rho_\mu (t=0) = \frac{1}{2}(1 + \sigma_\mu^{\hat n}),
```

Comment:
So I only noticed this here, we're missing a `\mathbb` around 1 (to indicate the identity matrix) that we have on the following lines...

But this doesn't seem to alter numerics: https://tex.stackexchange.com/questions/26637/how-do-you-get-mathbb1-to-work-characteristic-function-of-a-set (and it doesn't in my VSCode MD viewer).

In other places it seems like we're using `\mathbb{I}`: https://github.com/muon-spectroscopy-computational-project/muspinsim/blob/573c0065c8c01211e5fbf17a9b8e860c65094d92/docs/docs/theory_1.md?plain=1#L390

It might be that my formatter isn't working and this would appear OK in the actual docs, but if not I think it might be better to adopt `\mathbb{I}` everywhere (both in this PR and the recent Celio changes where I unfortunately didn't pick up on this).

Reply:
@patrick-austin 
It should work on the site, I run using `mkdocs serve` where it looks like
![image](https://user-images.githubusercontent.com/90245114/206467234-a87c9cf4-1cd2-4e80-b5a7-c3b63609b1a4.png)
I think its a matter of preference, but \mathbb{I} is only used twice and I find it more confusing
![image](https://user-images.githubusercontent.com/90245114/206467622-64c0dcae-a18d-4568-9251-b1f9589b1761.png)

So perhaps its worth just changing the two of them to \mathbb{1}?

Reply:
I have just edited it to change them all to \mathbb{1}.

Reply:
I'm now serving the docs locally and you're right, seems like it's just VSCode that doesn't format the numerics, and the documentation itself is good with `\mathbb{1}`. I agree that given the option between double stroke 1 and I, 1 is better for representing the identity matrix and we should use that consistently throughtout.

Reply:
@joelvdavies Maybe I'm misreading this but should if this is the density **matrix** should we not also be using the symbol for identity here? Like we do on line 81?
```suggestion
 \rho_\mu (t=0) = \frac{1}{2}(\mathbb{1} + \sigma_\mu^{\hat n}),
```